app/v2/models/responses/MessageSummary.scala

```diff
@@ -35,14 +36,16 @@ object MessageSummary {
       (JsPath \ "received").read[OffsetDateTime] and
       (JsPath \ "messageType").read[MessageType] and
       (JsPath \ "body").readNullable[XmlPayload] and
-      (JsPath \ "status").readNullable[MessageStatus])(MessageSummary.apply _)
+      (JsPath \ "status").readNullable[MessageStatus] and
```

Comment:
Json.Format[MessageSummary] ?

Reply:
My bad, we do need it as we need to know which Payload type to use for the reads (JsPath \ "body").readNullable[XmlPayload]